test/collection_test.js

```diff
@@ -610,6 +610,49 @@ describe("Collection", () => {
           last_modified: remote.last_modified
         });
     });
+
+    it("should ignore resolved conflicts during sync", () => {
```

Comment:
nitpick: since this test applies to `pullChanges()` and not `resolve()` it should probably be moved.